processes/third_party_triage_guidelines.md

```diff
@@ -0,0 +1,49 @@
+# Node.js Ecosystem Triage Guidelines
+
+## Introduction
+
+The purpose of this document is to bring the Node.js Ecosystem Triage team to a common understanding of how to triage and disclose security vulnerabilities reported through the Node.js third-party modules program hosted on HackerOne (https://hackerone.com/nodejs-ecosystem).
+
+This is a living document and is expected to evolve over time.
```

Comment:
I don't know this is worth saying, all our docs evolve over time!

Reply:
This was copied from its original Google Doc format, I will remove this line.